src/OrchardCore.Modules/OrchardCore.Resources/Assets.json

```diff
@@ -55,16 +55,36 @@
   {
     "generateSourceMaps": false,
     "inputs": [
-      "node_modules/trumbowyg/dist/ui/sass/trumbowyg.scss",
-      "node_modules/trumbowyg/plugins/*/ui/sass/*.scss"
+      "node_modules/trumbowyg/dist/ui/sass/trumbowyg.scss"
     ],
     "output": "wwwroot/Styles/trumbowyg.css"
   },
+  {
+    "generateSourceMaps": false,
+    "inputs": [
+      "node_modules/trumbowyg/dist/plugins/*/ui/sass/*.scss"
+    ],
+    "output": "wwwroot/Styles/trumbowyg-plugins.css"
+  },
   {
     "inputs": [
-      "Assets/trumbowyg/js/trumbowyg.js",
-      "Assets/trumbowyg/plugins/*/*.js"
+      "node_modules/trumbowyg/dist/trumbowyg.js"
```

Comment:
Do we need to also copy the Licence file in the wwwroot folder that contains the generated files?